tests/webdriver-e2e/specs/login/login.spec.js

```diff
@@ -24,17 +25,34 @@ describe('Login and logout tests', () => {
     user: 'Nombre de usuario',
     pass: 'Contraseña',
     error: 'Nombre de usuario o contraseña incorrecto. Favor intentar de nuevo.'
-  };   
+  };
+
+  afterEach(async () => {
+    await browser.deleteCookies();
+    await browser.refresh();
+  });
 
   it('should show locale selector on login page', async () => {
-    await commonElements.goToLoginPageNative();
-    const locales = await loginPage.getAllLocales(); 
-    expect(locales).toEqual(defaultLocales);  
+    const locales = await loginPage.getAllLocales();
```

Comment:
How come we're already on the login page? I think we should assume that this isn't the first test we run and that the browser is likely on some other page logged in with a test account.

Reply:
WDIO treats each file as a new browser session. Thus it will always be on the login page at the beginning of each test with a fresh browser. This is preferred. We have to do basically 0 browser management between test files other than log in with the proper account for your test file. As we are going to introduce more offline tests we won't have to play the log out login game or clearing data in the browser as we do with our protractor suite.

Reply:
Ah, nice!